libmscore/measure.cpp

```diff
@@ -2799,8 +2799,8 @@ Measure* Measure::cloneMeasure(Score* sc, int tick, TieMap* tieMap)
       m->_len         = _len;
       m->_repeatCount = _repeatCount;
 
-      Q_ASSERT(sc->staves().size() >= _mstaves.size()); // destination score must we're cloning into have at least as many staves as measure being cloned
-
+       Q_ASSERT(sc->staves().size() >= int(_mstaves.size())); // destination score we're cloning into must have at least as many staves as measure being cloned
+       
```

Comment:
Sorry to do this, but now there is an extra space in front of Q_ASSERT, and the next line has a bunch of unnecessary spaces as well.

Also, are you planning on squashing your commits? Are you waiting for more review first?

Reply:
@mattmcclinch I gave this to a friend to test and he discovered one more issue I'm trying to take care of: pickup measures that if at the beginning of repeat should be merged to the last measure of the previous repeat section.

Will fix the Q_ASSERT in the same commit and then squash all of them for easier review.